src/Templates/NewModule/build.psake.ps1

```diff
@@ -375,6 +410,7 @@ function PromptUserForCredentialAndStorePassword {
 }
 
 function AddSetting {
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSShouldProcess', '', Scope='Function')]
```

Comment:
As a matter of preference, I would put this outside the function.  But that might just be the C# dev in me.